flutter/android/android-docker.mk

```diff
@@ -30,8 +32,6 @@ flutter_common_docker_flags= \
 		--rm \
 		${flutter_docker_tty_arg} \
 		--init \
-		--user `id -u`:`id -g` \
-		--env USER=mlperf \
```

Comment:
@d-uzlov 
1. Do you know why we need `--user ...` here? The docker container is sandboxed so using root would be ok and can avoid some permission problem, though.
2. The `--env USER=mlperf` seems to be from legacy code, I don't see it used anywhere.

Reply:
We mount a host volume into docker.
If the program in docker creates any files in the mounted volume, those files will be owned by root.
I don't remember exactly how much problems this brings, but at least all build files would probably require sudo to delete or change them.

>The `--env USER=mlperf` seems to be from legacy code, I don't see it used anywhere.

As far am I'm aware, currently the only thing it does is it's removes "unknown user" warning if you open the terminal inside the docker image.

Reply:
Ok. I added it back. I tried to cache Bazel output so I need to remove it else the docker container cannot access the file (see https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#user). But caching Bazel output makes the scan command fail, so no need to remove it now.